### PR TITLE
Cherry-picks for the 4.2 branch (future 4.2.2) - 2nd batch

### PR DESCRIPTION
**Pull request by YuriSizov**
The focus is on crashes, regressions, and bugfixes in general, as well as documentation improvements. There are still more PRs available, but they'll have to wait until 4.2.3.

### CODE REVIEW

**Comment by AThousandShips** [comment]
Nite that https://github.com/godotengine/godot/pull/87114 depends on both of:
* https://github.com/godotengine/godot/pull/87118
* https://github.com/godotengine/godot/pull/87264

File: tests/test_main.cpp [hunk 1]
```diff
@@ -121,10 +117,18 @@
 #include "tests/scene/test_window.h"
 #include "tests/servers/rendering/test_shader_preprocessor.h"
 #include "tests/servers/test_navigation_server_2d.h"
-#include "tests/servers/test_navigation_server_3d.h"
 #include "tests/servers/test_text_server.h"
 #include "tests/test_validate_testing.h"
 
+#ifndef _3D_DISABLED
+#include "tests/scene/test_camera_3d.h"
```
Comment (AThousandShips) [inline tests/test_main.cpp:124]:
```suggestion
```
This test isn't in 4.2

Comment (YuriSizov) [reply tests/test_main.cpp]: Yep.

**Review by akien-mga — APPROVED** [review]
Looks good at a glance.

**Pull request merged**